finetune/target_models/document_labeling.py

```diff
@@ -0,0 +1,140 @@
+from finetune.target_models.sequence_labeling import SequenceLabeler
+
+def get_context(document, dpi_norm):
+    context = []
+    for page in document:
+        for token in page["tokens"]:
+            pos = token["position"]
+            offset = token["doc_offset"]
+            if dpi_norm:
+                dpi = page["pages"][0]["dpi"]
+                x_norm = 300 / dpi["dpix"]
+                y_norm = 300 / dpi["dpiy"]
+            else:
+                x_norm = 1.
+                y_norm = 1.
+
+            context.append(
+                {
+                    'top': pos["top"] * y_norm,
+                    'bottom': pos["bottom"] * y_norm,
+                    'left': pos["left"] * x_norm,
+                    'right': pos["right"] * x_norm,
+                    'start': offset["start"],
+                    'end': offset["end"],
+                    'text': token["text"],
+                }
+	    )
+    return context
+
+def reblock_by_offsets(offsets, texts, to_reblock):
```

Comment:
quick docstring here? Something like "update the start and end indices of the text so that individual examples always have a start index of 0, regardless of where they originally occurred in the document."

also, I wonder if "reblock" is a bit misleading? since we're not actually changing the relatively ordering or blocking?